From 022f8944ef2116732642682b68fce787ab9325da Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:06:29 -0400
Subject: [powerpc] select ARCH_MIGHT_HAVE_PC_PARPORT

Message-id: <1410545655-205645-161-git-send-email-dzickus@redhat.com>
Patchwork-id: 94309
O-Subject: [RHEL7 PATCH 160/626] powerpc: select ARCH_MIGHT_HAVE_PC_PARPORT
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit b7e7c37b6e84bd1aec8085606fa3fb42b5cf5a66
Author: Mark Salter <msalter@redhat.com>
Date:   Mon Oct 7 22:15:32 2013 -0400

    powerpc: select ARCH_MIGHT_HAVE_PC_PARPORT

    Architectures which support CONFIG_PARPORT_PC should select
    ARCH_MIGHT_HAVE_PC_PARPORT.

    Signed-off-by: Mark Salter <msalter@redhat.com>
    CC: Benjamin Herrenschmidt <benh@kernel.crashing.org>
    CC: Paul Mackerras <paulus@samba.org>
    CC: linuxppc-dev@lists.ozlabs.org
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/Kconfig b/arch/powerpc/Kconfig
index 763f63d..f90906c 100644
--- a/arch/powerpc/Kconfig
+++ b/arch/powerpc/Kconfig
@@ -85,6 +85,7 @@ config GENERIC_HWEIGHT
 config PPC
  bool
  default y
+ select ARCH_MIGHT_HAVE_PC_PARPORT
  select BINFMT_ELF
  select OF
  select OF_EARLY_FLATTREE
-- 
1.7.1